apps/demos/testing/widgets/fileuploader/CustomDropzone.test.js

```diff
@@ -0,0 +1,63 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { Selector, ClientFunction } from 'testcafe';
+import { runManualTest } from '../../../utils/visual-tests/matrix-test-helper';
+import { testScreenshot } from '../../../utils/visual-tests/helpers/theme-utils';
+
+const DROPZONE_EXTERNAL_CLASS = 'dropzone-external';
```

Comment:
As I see this is not a CSS class but ID of an HTML element. Can we change the name of variable depending on the kind of value it contains?